Source/Controllers/SubviewControllers/SPTablesList.m

```diff
@@ -245,6 +248,9 @@ - (IBAction)updateTables:(nullable id)sender
 				if (tableName == nil || [tableName isNSNull]) {
 					tableName = [eachRow objectForKey:@"NAME"];
 				}
+				if ((tableName == nil || [tableName isNSNull]) && eachRow.allValues.count == 1) {
```

Comment:
Should each of these if blocks be elseif instead of if?

Reply:
Also, right now aren't all 3 if blocks all triggering if tableName == nil || [tableName isNSNull]? Seems like a minor refactor may be in order here

Reply:
Yes because in every single one we try to assign the value... it could be improved by checking the `objectForKey` itself, this just seems more safe for now